tensorflow/lite/testing/model_coverage/model_coverage_lib.py

```diff
@@ -83,7 +83,8 @@ def _convert(converter, **kwargs):
   Args:
     converter: TFLiteConverter object.
     **kwargs: Additional arguments to be passed into the converter. Supported
-      flags are {"target_ops", "post_training_quantize", "quantize_to_float16"}.
+      flags are {"target_ops", "post_training_quantize",
+       "quantize_to_float16", "quant_16x8", "model_input_size"}.
```

Comment:
Nit: Can we at least call this "quantize_16x8" or "post_training_quantize_16x8" for consistency with other "quantize" flags?